Source/WebCore/css/ComputedStyleExtractor.cpp

```diff
@@ -744,6 +744,12 @@ static Ref<CSSValue> computedTransform(RenderElement* renderer, const RenderStyl
         return list;
     }
 
+    // https://w3c.github.io/csswg-drafts/css-transforms-1/#serialization-of-the-computed-value
+    // If we don't have a renderer, then the value should be "none" if we're asking for the
+    // resolved value (such as when calling getComputedStyle()).
+    if (valueType == ComputedStyleExtractor::PropertyValueType::Resolved)
+        return cssValuePool.createIdentifierValue(CSSValueNone);
```

Comment:
Does this problem exist with other properties that require reading from render tree?

Reply:
I don't know but I will look into other cases where we check for a `nullptr` renderer as indication that we're looking for the computed value vs. the resolved value. We should probably look at all the functions where the `renderer` is checked for `nullptr` and determine if we should have a check similar to what was added here.

Reply:
Right